packages/ui/src/components/editor/panels/Viewport/container/index.tsx

```diff
@@ -165,8 +165,8 @@ const ViewPortPanelContainer = () => {
 
   return (
     <ViewportDnD>
-      <div className="relative z-30 flex h-full w-full flex-col bg-theme-surface-main">
```

Comment:
change the css variable instead of introducing hardcoded values here

Reply:
The background color (`bg-theme-surface-main`) is already defined in the tailwind config. Rest of them are tailwind classes. Can you elaborate which hard-coded values are you referring to?

Reply:
Done